tests/integration/components/lt-head-test.js

```diff
@@ -62,6 +63,35 @@ test('click - sortable column', function(assert) {
   click(sortableHeader);
 });
 
+test('render sort icons', function(assert) {
+  this.set('table', new Table(Columns));
+
+  this.render(hbs`{{lt-head table=table renderInPlace=true iconSortable='fa-sort' iconAscending='fa-sort-asc' iconDescending='fa-sort-desc'}}`);
+
+  let allHeaders = findAll('tr > th ');
+  let sortableHeader = allHeaders[allHeaders.length - 1];
+  let sortIcon = find('.lt-sort-icon', sortableHeader);
+
+  // Sortable case
+  assert.ok(hasClass(sortIcon, 'fa-sort'), 'Sortable icon renders');
```

Comment:
Before, if the column was sortable but hadn't been sorted yet, it would display the icon in the DOM but with just the `lt-sort-icon` class and no other classes (`<i class="lt-sort-icon"></i>`).

To get that same result now, you would just not pass in any `iconSortable` to `lt-head` (because by default `iconSortable` is `''`).  So it should probably be stated somewhere that it is an optional field that you shouldn't pass in if you don't want an icon for flagging that the column is sortable.

I could also try to put together a Cookbook example of an `iconSortable` that only shows on hover.  Let me know if you think that would be helpful!